### ISSUE TITLE
text encoding for Socrata

### ISSUE THREAD

**Issue by tcaddy** [codeforgso/LOE_dashboard#19]
The raw data in the Socrata dataset is not being properly encoded during data exports.  For example, look at the case notes for this record: https://data.greensboro-nc.gov/Government/bad-encoding/25rz-wgyf

You will see something like `it?s` instead of `it's`.  We need to properly encode the text to fix these characters that show up as question marks.

**Comment by tcaddy** [comment]
Possible solution in Powershell: http://superuser.com/q/397890

**Comment by tcaddy** [comment]
See case_notes for this case number: 201101112

**Comment by tcaddy** [comment]
Looks like the problem is still persistent after last night's data refresh.

Here is a link to a [screenshot of the problem](http://imgur.com/a/29eR4).

**Comment by cjevans1123** [comment]
The PowerShell script which exports LOE data from the City into Socrata was edited Wednesday evening (the script had already ran Wednesday before this fix). The script ran again last night with the encoding fix for the CSV output.  I checked the Nuisance case 201101112 this morning to make sure and the issue has been fixed. Thanks for noticing this and helping us refine our automation processes for Open data, it's (or it?s) much appreciated!
